### PR TITLE
frmw1374-backportdeprecated

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1374

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
FYI:

BEFORE | AFTER
----- | -----
**Apply** | `rxApply`
**Auth** | `rxIdentity` or `rxSession`
**devicePaths** | `rxDevicePaths`
**Environment** | `rxEnvironment`
**ErrorFormatter** | `rxErrorFormatter`
**feedbackApi** | `rxFeedbackApi`
**feedbackTypes** | `rxFeedbackTypes`
**Identity** | `rxIdentity`
**NotifyProperties** | `rxNotifyProperties`
**PageTracking** | `rxPageTracker`
**Page** | `rxPager`
**Paginate** (filter) | `rxPaginate` (filter)
**Permission** | `rxSession`
**rxAuth** | `rxIdentity` or `rxSession`
**SelectFilter** | `rxSelectFilter`
**Session** | `rxSession`
**SessionStorage** | `rxSessionStorage`
**Status** | `rxStatus`
**StatusUtil** | `rxStatus`
**StatusUtil.setupScope()** | `rxStatus.setScope()`
**titleize** | `rxTitleize`
**TokenInterceptor** | `rxTokenInterceptor`
**UnauthorizedInterceptor** | `rxUnauthorizedInterceptor`
**urlUtils** | `rxUrlUtils`
**UtcOffsets** | `rxUtcOffsets`
**xor** | `rxXor`

File: src/utilities/helperClasses/docs/examples/helperClasses.simple.html [hunk 1]
```diff
@@ -1,31 +0,0 @@
-<h1>My Items <span class="subdued">(10)</span></h1>
```
Comment (CITguy) [inline src/utilities/helperClasses/docs/examples/helperClasses.simple.html]: Please put `src/utilities/helperClasses` files back.

Comment (cathysiller) [inline src/utilities/helperClasses/docs/examples/helperClasses.simple.html]: `helperClasses` put back into utilities

**Comment by jj4th** [comment]
Doesn't appear to affect midways, but I looked it over anyway.  Looks good to me :smile_cat:

**Comment by jj4th** [comment]
I can't be certain if this affects visreg... does the `#/Breadcrumbs` url for example still work?

**Comment by CITguy** [comment]
@jj4th: None of the paths were changed in this PR, so whatever paths were being used in visreg should still work.

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**